src/main/java/core/WebDriverManagerSingleton.java

```diff
@@ -0,0 +1,49 @@
+package core;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.edge.EdgeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+
+/**
+ * This class defines WebDriver.
+ */
+public class WebDriverManagerSingleton {
+    public static WebDriver driver;
+
+    /**
+     * Gets an instance of WebDriver according to browser.
+     *
+     * @param browser enum with the browser name.
+     * @return an instance of WebDriver
+     */
+    public static WebDriver getWebDriver(Browser browser) {
+        if (driver == null) {
+            if (browser == Browser.CHROME) {
```

Comment:
Maybe it would be better to try other options than conditional if, it could be set in another class

Reply:
Yes, you are right. This is something that I plan on doing in a future PR